spec/Section 3 -- Type System.md

```diff
@@ -2033,19 +2033,20 @@ directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 Note: The order in which directives appear may be significant, including
 repeatable directives.
 
-**Validation**
+**Type Validation**
 
-1. A directive definition must not contain the use of a directive which
+1. A Directive definition must include at least one DirectiveLocation.
```

Comment:
**_Why is this worded differently from the other "1 or more" constraints?_**
If this were worded in the same style as Objects, it would read "A Directive must define one or more DirectiveLocations"

To be ultra pedantic, while this sentence attempts to use a plural form of the DirectiveLocation token, it can also be interpreted as a plural form of DirectiveLocation**s** tokens, which has a distinct definition in Appendix B.4.

I thought that this phrasing was less ambiguous, though recognize there's also value in using consistent language.  I'm open to feedback.

Reply:
We're also generally okay with `must include one or more DirectiveLocation`.